src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.util.Arrays;
+
 public class FileWork {
+
     public String[] readFromFile(String fileName) {
-        return null;
+        String[] returnValue = new String[0];
+        File file = new File(fileName);
+        StringBuilder readerForWords = new StringBuilder();
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
+            StringBuilder stringBuilderForFile = new StringBuilder();
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                stringBuilderForFile.append(value).append(" ");
+                value = bufferedReader.readLine();
+            }
+            String[] workList = stringBuilderForFile.toString().split("\\s+");
```

Comment:
```suggestion
            String[] workList = stringBuilderForFile.toString().split("\\W+");
```

Reply:
Can you tell please, why?)

Reply:
this means split by any nonword character. You don't have to replace them later this way